.circleci/config.yml

```diff
@@ -41,24 +41,15 @@ jobs:
       - run:
           name: Test manylinux AArch64 wheel
           command: |
-
-            # install mpi dependencies
             sudo apt update
+            echo "deb http://ppa.launchpad.net/deadsnakes/ppa/ubuntu jammy main" | sudo tee /etc/apt/sources.list.d/deadsnakes-ppa.list && sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F23C5A6CF475977595C89F51BA6932366A755776 && sudo apt update
```

Comment:
I couldn't get `pyenv` to work with 3.13, so instead I've decided to use the [deadsnakes PPA](https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa) instead.